tests/lax_numpy_reducers_test.py

```diff
@@ -866,5 +866,18 @@ def testCumulativeSumBool(self):
                              dtype=jnp.bool_)
     np.testing.assert_array_equal(np.array([[True], [True], [False]]), out)
 
+  @jtu.sample_product(
+    [dtype for dtype in int_dtypes + unsigned_dtypes],
```

Comment:
Try something like this instead:
```suggestion
    dtype=[*int_dtypes, *unsigned_dtypes]
```

Reply:
After the fix, the tests work on my local device.